interactions/client/context.py

```diff
@@ -38,12 +38,6 @@ class _Context(ClientSerializerMixin):
     that the user-facing API is able to allow developers to
     easily access information presented from any event in
     a "contextualized" sense.
-
-    :ivar Optional[Message] message: The message data model.
-    :ivar Member author: The member data model.
-    :ivar User user: The user data model.
-    :ivar Optional[Channel] channel: The channel data model.
-    :ivar Optional[Guild] guild: The guild data model.
```

Comment:
keep it

Reply:
I added these ivars to Command & Component Contexts.
Also _Context is hidden now so why we need to keep it?

Reply:
i mean users can see it but if you think its not needed leave it out